src/introduction/introduction.tex

```diff
@@ -2,27 +2,51 @@
 \chapter{Introduction}\label{ch:introduction}
 % textidote: ignore end
 
-Learning is something we do every day.
-It is something most people enjoy and strive to experience.
-However, learning can also be difficult.
-In this report we will explore the concept of collaborated assisted learning and how software can assist in this method.
-We will be focusing our analysis around the age-old game of chess.
-
 Chess has seen a rise in player count in recent years, and with the growing player base the current chess websites have
-perfected their craft, this has been done through intuitive design and the time to implement extra features in the apps.
-One reason that chess has become more popular online, is that using the popular web apps like \url{chess.com} or
+perfected their craft, this has been done through intuitive design and the addition of extra features in the apps.
+One reason that chess has become more popular online, is that using popular web apps like \url{chess.com} or
 \url{lichess.org} allows the player to use a powerful tool to aid in their learning journey, that is chess engines.
 
+We believe that the utilization of chess engines is not fully satisfied yet, as engines are reserved for single player
+applications.
+
+% textidote: ignore begin
+\begin{figure}[htb]
+    \centering
+    \includegraphics[width=1\textwidth]{overview}
+    \caption{Understanding the project}\label{fig:project-overview}
+\end{figure}
+% textidote: ignore end
+
 Chess engine popularity has risen greatly in the past years and have been developed to a point that they
 outperform humans.
 These tools are essential to understanding the complexity of the end- and mid-game strategies of chess.
+This makes engines an optimal tool for teaching players the intricacies of chess which is exactly what
+differentiates the average chess players from masters.
+
+
 Chess encapsulates the spirit of a game that is easy to learn, but hard to master.
+That is why those who wish to become skilled in chess seek help from other masters in the form of a coach.
+Coaching is a well respected form of learning, but it has flaws that place barriers in terms of who can get this form of
+help.
+
+The biggest obstacle coaches have is the monetary aspect as most players do not have the money to afford a chess coach
+for extended periods of time.
+Coaches also need to have allocated time for their lessons which mean that it makes it more difficult to have the
+coaching sessions.
```

Comment:
This is more problem analysis than introduction. It is a great point and is well written but is better suited in the problem analysis IMO

Reply:
Andres' comments:
"This is the proposition value of your project. You need to expand on this. Motivate why is important. Briefly highlight the challenges and your solution. Include a rich figure. "

I think this could be called our solution to the highlighted challenges and solution. I see your point, but this is more of a balancing act and introducing the problems faced.

Reply:
I see. I am ok with keeping it in :)